t/16-markdown.t

```diff
@@ -128,33 +136,57 @@ subtest 'unsafe HTML filtered out' => sub {
+      'right markdown';
+    is bugref_to_html('brc#9876'),
+      '<a href="https://bugzilla.redhat.com/show_bug.cgi?id=9876" title="Bug referenced: brc#9876">brc#9876</a>',
+      'right markdown';
+    is bugref_to_html('bko#9876'),
+      '<a href="https://bugzilla.kernel.org/show_bug.cgi?id=9876" title="Bug referenced: bko#9876">bko#9876</a>',
       'right markdown';
-    is bugref_to_markdown('bgo#9876'), '[bgo#9876](https://bugzilla.gnome.org/show_bug.cgi?id=9876)', 'right markdown';
-    is bugref_to_markdown('brc#9876'), '[brc#9876](https://bugzilla.redhat.com/show_bug.cgi?id=9876)', 'right markdown';
-    is bugref_to_markdown('bko#9876'), '[bko#9876](https://bugzilla.kernel.org/show_bug.cgi?id=9876)', 'right markdown';
-    is bugref_to_markdown('poo#9876'), '[poo#9876](https://progress.opensuse.org/issues/9876)', 'right markdown';
-    is bugref_to_markdown('gh#foo/bar#1234'), '[gh#foo/bar#1234](https://github.com/foo/bar/issues/1234)',
+    is bugref_to_html('poo#9876'),
+      '<a href="https://progress.opensuse.org/issues/9876" title="Bug referenced: poo#9876">poo#9876</a>',
       'right markdown';
-    is bugref_to_markdown('kde#9876'), '[kde#9876](https://bugs.kde.org/show_bug.cgi?id=9876)', 'right markdown';
-    is bugref_to_markdown('fdo#9876'), '[fdo#9876](https://bugs.freedesktop.org/show_bug.cgi?id=9876)',
+    is bugref_to_html('gh#foo/bar#1234'),
+      '<a href="https://github.com/foo/bar/issues/1234" title="Bug referenced: gh#foo/bar#1234">gh#foo/bar#1234</a>',
       'right markdown';
-    is bugref_to_markdown('jsc#9876'), '[jsc#9876](https://jira.suse.de/browse/9876)', 'right markdown';
-    is bugref_to_markdown('pio#foo#1234'), '[pio#foo#1234](https://pagure.io/foo/issue/1234)', 'right markdown';
-    is bugref_to_markdown('pio#foo/bar#1234'), '[pio#foo/bar#1234](https://pagure.io/foo/bar/issue/1234)',
+    is bugref_to_html('kde#9876'),
+      '<a href="https://bugs.kde.org/show_bug.cgi?id=9876" title="Bug referenced: kde#9876">kde#9876</a>',
       'right markdown';
-    is bugref_to_markdown('ggo#GNOME/foo#1234'),
-      '[ggo#GNOME/foo#1234](https://gitlab.gnome.org/GNOME/foo/issues/1234)', 'right markdown';
-    is bugref_to_markdown('gfs#flatpak/fedora-flatpaks#26'),
-      '[gfs#flatpak/fedora-flatpaks#26](https://gitlab.com/fedora/sigs/flatpak/fedora-flatpaks/issues/26)',
+    is bugref_to_html('fdo#9876'),
+      '<a href="https://bugs.freedesktop.org/show_bug.cgi?id=9876" title="Bug referenced: fdo#9876">fdo#9876</a>',
       'right markdown';
-    is bugref_to_markdown("boo#9876\n\ntest boo#211\n"),
-      "[boo#9876](https://bugzilla.opensuse.org/show_bug.cgi?id=9876)\n\n"
-      . "test [boo#211](https://bugzilla.opensuse.org/show_bug.cgi?id=211)\n",
+    is bugref_to_html('jsc#9876'),
+      '<a href="https://jira.suse.de/browse/9876" title="Bug referenced: jsc#9876">jsc#9876</a>', 'right markdown';
```

Comment:
please use unique test description strings explicit about the individual checks you done.